eo-runtime/src/test/java/EOorg/EOeolang/EOio/EOstdinTest.java

```diff
@@ -47,11 +48,17 @@
  */
 @SuppressWarnings("JTCOP.RuleAllTestsHaveProductionClass")
 final class EOstdinTest {
+
+    /**
+     * Next line.
+     */
+    private static final String NEXT_LINE = "next-line";
```

Comment:
@c71n93 the same here -- don't share literals between tests, use unique ones for each test

Reply:
@yegor256 Here "next-line" is the name of EO attribute that is used in tests, so we can't use different strings in this case.